benchmarks/kubectl-mtb/pkg/benchmark_suite/benchmarks_suite.go

```diff
@@ -29,5 +33,38 @@ func (bs *BenchmarkSuite) ProfileLevel(pl int) []*benchmark.Benchmark {
 			benchmarksArray = append(benchmarksArray, b)
 		}
 	}
-	return benchmarksArray
+	sortedBenchmarks := SortBenchmarks(benchmarksArray)
```

Comment:
does this need to be public, or can it be package private?

Reply:
It can be private, I have changed it.